utilities/sell-remaining-coins.py

```diff
@@ -16,6 +16,19 @@
     load_correct_creds
 )
 
+from colorama import init
+init()
+
+# for colourful logging to the console
+class txcolors:
+    BUY = '\033[92m'
+    WARNING = '\033[93m'
+    SELL_LOSS = '\033[91m'
+    SELL_PROFIT = '\033[32m'
+    DIM = '\033[2m\033[35m'
+    DEFAULT = '\033[39m'

```

Comment:
We could move these colors to some `config` or `common` file so we can reuse on other files. What do you say, devs?